cdk/l2-cdk-resources/third-party-integration/.projenrc.js

```diff
@@ -0,0 +1,35 @@
+const { awscdk } = require('projen');
+const { ReleaseTrigger } = require('projen/lib/release');
+const thirdPartyIntegration = new awscdk.AwsCdkConstructLibrary({
+  author: 'MongoDBAtlas',
+  authorAddress: 'https://mongodb.com',
+  authorName: 'MongoDBAtlas',
+  cdkVersion: '2.1.0',
+  defaultReleaseBranch: 'INTMDB-548',
+  majorVersion: 1,
+  releaseToNpm: true,
+  releaseTrigger: ReleaseTrigger.manual(),
+  docgen: true,
+  name: '@mongodbatlas-awscdk/atlas-integrations',
+  repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+  sampleCode: false,
+  keywords: ['cdk',
+    'awscdk',
+    'aws-cdk',
+    'cloudformation',
+    'cfn',
+    'extensions',
+    'constructs',
+    'cfn-resources',
+    'cloudformation-registry',
+    'l2',
+    'mongodb',
+    'atlas',
+    'third-party-integration'],
+  description: 'Returns, adds, edits, and removes third-party service integration configurations. MongoDB Cloud sends alerts to each third-party service that you configure. Currently supporting MicrosoftTeams, Datadog and PagerDuty for a given Atlas project',
+  stability: 'experimental',
+  deps: ['@mongodbatlas-awscdk/third-party-integration@1.0.1'],
+  devDeps: ['@mongodbatlas-awscdk/third-party-integration@1.0.1'],
+  peerDeps: ['@mongodbatlas-awscdk/third-party-integration@1.0.1'],
+});
+thirdPartyIntegration.synth();
```

Comment:
👀

Reply:
I think all the tools aren't adding newlines, which is a little annoying.

Also, do we need to commit the generated code or can that be generated via the makefile as part of scripting when compiling/testing/publishing?

Better way of asking the question maybe, do we edit these files after the initial generation?

Reply:
yes, we have to edit `.projenrc.js` after it is generated

Reply:
For other generated files, I think since currently we publish manually, it's okay because we'll have tested code in the repo.

Reply:
updated new line

Reply:
For Typescript, I've installed Prettier for now in VS code, that should help in the future

Reply:
Nice. Not blocking this PR, but also check out how we enforce TS code formatting in MMS - it's a similar use-case we've solved.

A good candidate for a small improvement in the future.